GeneralsMD/Code/GameEngine/Include/GameClient/InGameUI.h

```diff
@@ -772,6 +776,31 @@ friend class Drawable;	// for selection/deselection transactions
 	Color												m_gameTimeColor;
 	Color												m_gameTimeDropColor;
 
+	// Network Latency Counter
```

Comment:
Can also move these above system time. Keep order consistent across the systems.